src/test/java/com/api/jsonata4java/expressions/BasicExpressionsTests.java

```diff
@@ -691,24 +691,24 @@ public void eventAccessWithQuotes() throws Exception {
     @Test
     public void appAccessNoQuotes() throws Exception {
 
-        JsonNode intface = mapper.readTree("{\"a\":2}");
+        JsonNode intFace = mapper.readTree("{\"a\":2}");
```

Comment:
I couldn't tell if this was an intentional misspelling of `interface`. I personally prefer `interface_` or `iface` in that case. I'm assuming that it isn't here...